tokio-codec/src/framed.rs

```diff
@@ -122,6 +122,11 @@ impl<T, U> Framed<T, U> {
         &mut self.inner.get_mut().get_mut().1
     }
 
+    /// Returns a reference to the read buffer.
+    pub fn read_buffer(&self) -> &BytesMut {
```

Comment:
would it make sense to expose a `write_buffer` fn too?

Reply:
In my use case, I have a write buffer that is external to the Framed, I then flush that into the Framed instance.

At least for my use case, I know what is in the Framed write buffer and it is not interesting.

It _might_ be useful, do you want me to add it ?